integration-core/src/test/java/org/symphonyoss/integration/core/BaseIntegrationTest.java

```diff
@@ -99,6 +104,16 @@ public void testRegisterUserCertNotFound() {
     integration.registerUser(APP_TYPE);
   }
 
+  @Test(expected = CertificateNotFoundException.class)
```

Comment:
Why is the unit test and the class under test (BaseIntegration) in two different repos?  They are not even in the same package in the two repos.